### PR TITLE
Remove WindowDecorationsHelper, fix theme in properties window

### LINKED ISSUE #9911: Bug: Title bar close/minimize/maximize buttons color don't match the app theme
**Requirements**
- Title bar buttons should match the app theme rather than the system theme

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- For #9780 ("Window Decoration api isn't working on the properties windows (we should remove the relevant code before release)")
- Closes #9911

**Changes**
- Removed AppWindowTitleBar.IsCustomizationSupported() check, not necessary as of winappsdk 1.2 (titlebar API supported down to 1809)
- Removed WindowDecorationsHelper, uwp only + same reason as above
- Removed code for custom titlebar close button (it was commented), we now use the standard close button
- Fix wrong background in properties window when theme changes
- Fix crash when clicking "share", remove call to GetForCurrentView() (uwp only)
- Restore code for changing titlebar buttons on theme change
- Improve positioning of properties window (do not go out of screen bounds)

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Helpers/FilePropertiesHelpers.cs [hunk 1]
```diff
@@ -47,65 +47,51 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
 
             if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
```
Comment (yaichenbaum) [inline src/Files.App/Helpers/FilePropertiesHelpers.cs:48]: Can this if statement be removed? I don't think there is a scenario where this is not the case.

File: src/Files.App/UWPToWinAppSDKUpgradeHelpers.cs [hunk 2]
```diff
@@ -23,6 +23,9 @@ interface IDataTransferManagerInterop
 
     public static class InteropHelpers
     {
+        public static readonly Guid DataTransferManagerInteropIID =
+            new Guid(0xa5caee9b, 0x8708, 0x49d1, 0x8d, 0x36, 0x67, 0xd2, 0x5a, 0x8d, 0xa0, 0x0c);
```
Comment (yaichenbaum) [inline src/Files.App/UWPToWinAppSDKUpgradeHelpers.cs:27]: What does this do?

File: src/Files.App/Helpers/FilePropertiesHelpers.cs (hunk 1 above)
Comment (gave92) [reply src/Files.App/Helpers/FilePropertiesHelpers.cs:48]: Min version is 17763 which has UniversalApiContract 7

File: src/Files.App/UWPToWinAppSDKUpgradeHelpers.cs (hunk 2 above)
Comment (gave92) [reply src/Files.App/UWPToWinAppSDKUpgradeHelpers.cs:27]: Heck if I know :) Comes from the sample (it's the COM Guid of the DataTransferManager interface or something)

File: src/Files.App/Helpers/FilePropertiesHelpers.cs (hunk 1 above)
Comment (yaichenbaum) [reply src/Files.App/Helpers/FilePropertiesHelpers.cs:48]: I thought we were using 1809. We can keep this then 👍

Comment (gave92) [reply src/Files.App/Helpers/FilePropertiesHelpers.cs:48]: Yeah 1809 == 10.0.17763.0

**Comment by yaichenbaum** [comment]
> Restore code for changing titlebar buttons on theme change

@gave92 this seems to help for the main window, but the issue still exists with the property windows.

**Comment by gave92** [comment]
Should be good now.

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Comment by Fahhd1400** [comment]
دمج طلب السحب

**Pull request merged**